src/pipeline_blocks.py

```diff
@@ -440,12 +441,38 @@ def feature_extraction(config, train_mode, suffix, **kwargs):
         return feature_combiner
 
 
+def xgb_preprocessing(features, config, train_mode, suffix, **kwargs):
+    if train_mode:
+        features, features_valid = features
+
+    one_hot_encoder = Step(name='one_hot_encoder{}'.format(suffix),
+                           transformer=OneHotEncoder(ce.OneHotEncoder(
+                               **config.xgb_preprocessing.one_hot_encoder)
+                           ),
+                           input_steps=[features],
+                           adapter=Adapter({'X': E(features.name, 'features')}),
```

Comment:
@pknut it seems that this encoder is encoding the entire table and not just the categorical features. Am I right?